azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/IEventHubClient.java

```diff
@@ -51,6 +51,18 @@ PartitionReceiver createReceiverSync(String consumerGroupName, String partitionI
     CompletableFuture<PartitionReceiver> createReceiver(String consumerGroupName, String partitionId, String startingOffset, boolean offsetInclusive)
                                                     throws EventHubException;
 
+    PartitionReceiver createReceiverSync(String consumerGroupName, String partitionId, long startingSequenceNumber)
+            throws EventHubException;
+
+    CompletableFuture<PartitionReceiver> createReceiver(String consumerGroupName, String partitionId, long startingSequenceNumber)
+            throws EventHubException;
+
+    PartitionReceiver createReceiverSync(String consumerGroupName, String partitionId, long startingSequenceNumber, boolean sequenceNumberInclusive)
+            throws EventHubException;
+
+    CompletableFuture<PartitionReceiver> createReceiver(String consumerGroupName, String partitionId, long startingSequenceNumber, boolean sequenceNumberInclusive)
+            throws EventHubException;
```

Comment:
@sabeegrewal as discussed let's follow the same pattern that we did in other SDK and use EventPosition class for offset/seqneuceNumber/enqueue time filter and remove existing APIs - this change will be shipped in v1.0 release.